### PR TITLE
Aesthetic stuff

### PR DESCRIPTION
**Pull request by TdewT**
Several graphical improvements on frontend and in readme.

Merged with recent main pushes for easy PR resolve.
I didn't find any issues with the merge on my end, but it is advised to check if everything is working properly.

### CODE REVIEW

**Comment by TdewT** [comment]
Just FYI this branch is still in development, this branch is to be merged, but not deleted.

File: json_templates/api_tokens.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "tokens ": {
+    "this websites api tokens along with the IP from which they were created (used when resetting token)": {
+      "{users:IP:address}": "{token}"
```
Comment (Creeperasek) [inline json_templates/api_tokens.json:4]: Shouldn't here template be like this: {users:IPaddress} ? or address is a different value ?

**Review by Creeperasek — DISMISSED** [review]
In all this looks very good my only issue is with api token (comment) but if its correct then ignore the comment and merge

File: json_templates/api_tokens.json (hunk 1 above)
Comment (TdewT) [reply json_templates/api_tokens.json]: I tried to convey that IP is saved using ':' as divider between numbers.
It is now changed for a more clear phrase.

**Review by Creeperasek — APPROVED** [review]

**Pull request merged**